openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -233,22 +233,9 @@ Requirements for both the Wallet and the Verifier:
 This profile defines the following additional requirements for IETF SD-JWT VCs as defined in [@!I-D.ietf-oauth-sd-jwt-vc].
 
 * Compact serialization MUST be supported as defined in [@!I-D.ietf-oauth-selective-disclosure-jwt]. JSON serialization MAY be supported.
-* The following JWT Claims MUST be supported Content (differentiate issuance & presentation)
-
-| Claim | SD-JWT as issued by the Issuer | Normative Definition |
-|:--- |:--- |:--- |
-| iss | MUST |[@!RFC7519], Section 4.1.1 |
-| iat | MUST |[@!RFC7519], Section 4.1.6 |
-| exp | SHOULD (at the discretion of the Issuer) | [@!RFC7519], Section 4.1.4 |
-| cnf |	MUST |	[@!RFC7800]|
-| vct |	MUST | [@!I-D.ietf-oauth-sd-jwt-vc]|
-|status| SHOULD (at the discretion of the Issuer)| [@!I-D.ietf-oauth-status-list]|
-
-* The Issuer MUST NOT make any of the JWT Claims in the table above to be selectively disclosable, so that they are always present in the SD-JWT-VC presented by the Holder.
-* It is at the discretion of the Issuer whether to use `exp` claim and/or a `status` claim to express the validity period of an SD-JWT-VC. The Wallet and the verifier  MUST support both mechanisms.
-* The `iss` claim MUST be an HTTPS URL.
-* The `vct` JWT claim as defined in [@!I-D.ietf-oauth-sd-jwt-vc].
-* The `cnf` claim [@!RFC7800] MUST conform to the definition given in [@!I-D.ietf-oauth-sd-jwt-vc]. Implementations conforming to this profile MUST include the JSON Web Key [@!RFC7517] in the `jwk` sub claim.
+* It is at the discretion of the Issuer whether to use `exp` claim and/or a `status` claim to express the validity period of an SD-JWT VC. The Wallet and the Verifier MUST support both mechanisms.
```

Comment:
```suggestion
* It is at the discretion of the Issuer whether to use `exp` claim and/or a `status` claim to express the validity period of an SD-JWT VC. The Wallet and the Verifier MUST support both mechanisms.
```
This seems redundant but not a hill I would die on.

Reply:
I'd tend to agree, I think in particular that MUST is not covered elsewhere.

I am not sure if the current language means "issuer can choose to use neither exp nor status" or if it means "issuer has to use at least one of exp & status"? That might be best handled in a new issue though.

Reply:
Transferred this into an issue: https://github.com/openid/OpenID4VC-HAIP/issues/256